src/screens/Home/index.js

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Text, View, Button, StyleSheet, FlatList, Alert } from 'react-native'
+
+import Container from '../../components/Container'
+import http from '../../utils/http'
+
+export default class Home extends Component {
+  static propTypes = {
+    navigation: PropTypes.object.isRequired,
+  }
+
+  state = {
+    isLoading: true,
+    ages: [],
+    topics: [],
+  }
+
+  async componentDidMount () {
+    try {
+      const { data } = await http.get('https://my-json-server.typicode.com/gluix20/treasure/db')
+      this.setState({
+        isLoading: false,
+        ...data,
+      })
+    } catch (error) {
+      Alert.alert('Error', error.message)
+    }
+  }
+
+  onPress = () => {
+    this.props.navigation.navigate('Home2')
+  }
+
+  renderAgeItem = ({item}) => (
+    <View>
+      <Text style={localStyles.sectionHeader}>{item.text}</Text>
+      {this.getTopicsItems()}
+    </View>
+  )
+
+  renderTopicItem = ({item}) => (
+    <View>
+      <Text style={localStyles.item}>
+        {item.text}
+      </Text>
+    </View>
+
+  )
+
+  keyExtractor = (item, index) => {
+    return index.toString()
+  }
+
+  getAgesItems = () => (
+    <View>
+      <FlatList
+        data={this.state.ages}
+        renderItem={this.renderAgeItem}
+        keyExtractor={this.keyExtractor}
+      />
+    </View>
+  )
+
+  getTopicsItems = () => (
+    <View>
+      <FlatList
+        data={this.state.topics}
+        renderItem={this.renderTopicItem}
+        keyExtractor={this.keyExtractor}
+      />
+    </View>
+  )
+
+  render () {
+    const { isLoading } = this.state
+    return (
+      <Container isLoading={isLoading} style={localStyles.container}>
+        <Text style={[{textAlign: 'center', marginTop: 40}]}>HOME</Text>
+        <Button title='to Home-2' onPress={this.onPress} />
+        {/* <FlatList
+          data={ages}
+          renderItem={this.getAgesItems}
+          keyExtractor={(item, index) => index.toString()}
+        /> */}
+
+        {this.getAgesItems()}
+      </Container>
+    )
+  }
+}
+
+const localStyles = StyleSheet.create({
+  container: {
+    flex: 1,
```

Comment:
`Container` already has `flex` as 1, so I think there is no need for adding it here again

Reply:
I removed that style, to use only the Container component default style.